addOns/bugtracker/src/main/java/org/zaproxy/zap/extension/bugtracker/BugTrackerGithubConfigParams.java

```diff
@@ -24,29 +24,44 @@
 class BugTrackerGithubConfigParams extends Enableable {
 
     private String username;
-    private String password;
+    private String token;
     private String repoUrl;
 
     public BugTrackerGithubConfigParams() {
         this("", "", "");
     }
 
-    public BugTrackerGithubConfigParams(String username, String password, String repoUrl) {
+    public BugTrackerGithubConfigParams(String username, String token, String repoUrl) {
         this.username = username;
-        this.password = password;
+        this.token = token;
         this.repoUrl = repoUrl;
     }
 
     public BugTrackerGithubConfigParams(BugTrackerGithubConfigParams config) {
-        this(config.username, config.password, config.repoUrl);
+        this(config.username, config.token, config.repoUrl);
     }
 
     public String getName() {
         return username;
     }
 
+    /**
+     * @deprecated v4 use {@link #getToken()} instead
+     * @return The password to connect.
+     */
+    @Deprecated
```

Comment:
Not worth, the add-on is not using SemVer.

Reply:
Shall we switch it to semver? `4.0.0`?

Reply:
I don't think it's necessary at this point, I'd expect this to be used more with the UI than programmatically (e.g. `ExtensionBugTracker` does not expose an API to raise issues).

Reply:
I was suggesting not deprecate at all, just rename it.

Reply:
Oh okay, can do that too.